apiv6/examples/demos/retry_demo/main.go

```diff
@@ -10,8 +10,10 @@ import (
 	"net/url"
 	"time"
 
+	"github.com/kentik/community_sdk_golang/apiv6/kentikapi/httputil"
```

Comment:
Why does `httputil` need to be in `apiv6` specific directory?

Reply:
You're right and it doesn't need to be there. PRs #59, #60 and #61 are all part of one issue that is "Provide one entry-point to Kentik Go API library". There are three PRs to increase readability in during review. In PR #61 `httputil` is moved to a folder with kentikapi's internal packages.